src/ir/auth_logic/BUILD

```diff
@@ -91,3 +91,19 @@ cc_test(
         "//src/common/testing:gtest",
     ],
 )
+
+cc_library(
+    name = "ast_construction",
+    srcs = ["ast_construction.cc"],
+    hdrs = ["ast_construction.h"],
+    copts = ["-fexceptions"],
```

Comment:
Yes. Removing this would cause build to fail with folllowing error.
 (error: exception handling disabled, use '-fexceptions' to enable
   70 |     throw "there is no serialized ATN";)